tasks/django.yml

```
@@ -147,8 +147,8 @@
 
 - name: load initial data
   environment:
-    PRIVATE_KEY: "{{ default_private_ssh_key.stdout }}"
-    PUBLIC_KEY: "{{ default_public_ssh_key.stdout }}"
+    PRIVATE_KEY: "{{ default_private_ssh_key.stdout|default(None) }}"
+    PUBLIC_KEY: "{{ default_public_ssh_key.stdout|default(None) }}"
```

Shouldn't these be breezy? Or, is there a specific reason to write it like that? 

```suggestion
    PRIVATE_KEY: "{{ default_private_ssh_key.stdout | default(None) }}"
    PUBLIC_KEY: "{{ default_public_ssh_key.stdout | default(None) }}"
```